src/coord/tests/sql.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2018 sqlparser-rs contributors. All rights reserved.
+// Copyright Materialize, Inc. All rights reserved.
+//
+// This file is derived from the sqlparser-rs project, available at
+// https://github.com/andygrove/sqlparser-rs. It was incorporated
+// directly into Materialize on December 21, 2019.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License in the LICENSE file at the
+// root of this repository, or online at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{cell::RefCell, collections::BTreeMap, rc::Rc};
+
+use expr::GlobalId;
+use repr::{RelationDesc, RelationType};
+use tempfile::NamedTempFile;
+
+use coord::{
+    catalog::{Catalog, CatalogItem, Table},
+    session::Session,
+};
+use sql::{
+    ast::{Expr, Statement},
+    names::{DatabaseSpecifier, FullName},
+    plan::{resolve_names, PlanContext, QueryContext, QueryLifetime, StatementContext},
+};
```

Comment:
Could you configure Rust Analyzer to use "last" as its import merge behavior? That'll avoid making these deeply nested trees of `use`s and instead only use braces for the last path component, which is the import style we loosely try to enforce.